src/md/tekwill/topic5/Romb.java

```diff
@@ -0,0 +1,59 @@
+package md.tekwill.topic5;
+
+public class Romb extends FiguraGeometrica {
+    private double diagonalaMare;
+    private double diagonalaMica;
+
+    private double laturaMare;
+    private double laturaMica;
+
+
+    public void setDiagonale(double diagonalaMare, double diagonalaMica) {
+        if (diagonalaMare > diagonalaMica && diagonalaMica > 0 && diagonalaMare > 0) {
+            System.out.println("Valorile introduse sunt corecte");
+            this.diagonalaMare = diagonalaMare;
+            this.diagonalaMica = diagonalaMica;
+        } else {
+            System.out.println("Valoarea este incorecta");
+
+        }
+    }
+
+    public void setLaturi(double laturaMare, double laturaMica) {
+        if (laturaMare > laturaMica && laturaMare > 0 && laturaMica > 0) {
```

Comment:
for a better understanding, I would recommend to use () in order to delimit the operations: 
((laturaMare > laturaMica) && (laturaMare > 0) && (laturaMica > 0))